config/samples/azure_v1beta1_azuresqldatabase.yaml

```diff
@@ -9,4 +9,18 @@ spec:
   location: westus
   resourceGroup: resourcegroup-azure-operators
   server:  sqlserver-sample-777
-  edition: 0
\ No newline at end of file
+  edition: 0
+  # the following fields are optional:
+
+  # The weekly retention policy for an LTR backup in an ISO 8601 duration format (https://en.wikipedia.org/wiki/ISO_8601#Durations)
+  # weeklyRetention: P12W
+
+  # The monthly retention policy for an LTR backup in an ISO 8601 duration format
+  # monthlyRetention: P3M
+
+  # The yearly retention policy for an LTR backup in an ISO 8601 duration format
+  # yearlyRetention: P5Y
+
+  # The week of year to take the yearly backup, valid values [1, 52]
+  # weekOfYear: 16
```

Comment:
I added a bunch of examples as well as a link to the Wikipedia page which describes the format